README.md

```diff
@@ -135,3 +135,9 @@ Now, in your terminal:
      - Press Esc to close image window
 5. In `src/scripts/device_info.json`, add the `device_id` to the `device_cat` it belongs to
 6. Run `npx prettier src/scripts/device_info.json --write` to format `src/scripts/device_info.json`
+
+### About `src/pages/**/_*.ts`
+
+You might observe some js/ts files with underscroe `_` prefix. This is because ...
+
+ref https://docs.astro.build/en/guides/routing/#excluding-pages
```

Comment:
Can you help fine-tune this a bit? I was providing a template for you to modify 🤣